### ISSUE TITLE
Basketball website: `get_in` call when using a pipe is not detected

### ISSUE THREAD

**Issue by vecerek** [exercism/elixir-analyzer#330]
I'm receiving the following feedback:

> `Essential`
> Make sure to only use use `get_in/2` in `BasketballWebsite.get_in_path/2` and not in `BasketballWebsite.extract_from_path/2`. The purpose of this exercise is to practice using the Access Behaviour in two different ways, first directly using the `data[key]` syntax, and then indirectly using the `get_in_path/2`.

For the following code:
```ex
defmodule BasketballWebsite do
  def extract_from_path(data, path), do: do_extract_from_path(data, String.split(path, "."))
  defp do_extract_from_path(nil, _), do: nil
  defp do_extract_from_path(data, []), do: data
  defp do_extract_from_path(data, [head | tail]), do: do_extract_from_path(data[head], tail)

  def get_in_path(data, path), do: data |> get_in(String.split(path, "."))
end
```

I'm only using `get_in` inside `get_in_path/2`.

**Comment by angelikatyborska** [comment]
Thanks for the report! There is a bug in detecting that you did use `get_in` in `get_in_path`. For some reason, this code is correct:
```elixir
def get_in_path(data, path) do
  get_in(data, String.split(path, "."))
end
```

But this code makes the analyzer think `get_in` wasn't used:

```elixir
def get_in_path(data, path) do
  data |> get_in(String.split(path, "."))
end
```
